genomic-data-product.md

```diff
@@ -1,152 +1,163 @@
-### **Genome annotations**
-
-
+### Genome annotations
 
 Genome annotations are in [GFF format](http://useast.ensembl.org/info/website/upload/gff.html) GFF format contains genes and other genomic features. Allows “track” info for visualization [http:\/\/useast.ensembl.org\/info\/website\/upload\/gff.html](http://useast.ensembl.org/info/website/upload/gff.html)
 
-
-
-### **Visualizing and annotating Genomes**
-
-
+### Visualizing and annotating Genomes
 
 [Gbrowse](http://gmod.org/wiki/GBrowse) is a comprehensive database + interactive web application for manipulating and displaying annotations on genomes.
 
-
-
 # Genomics pipeline
```

Comment:
If we remove the above (since we don't use GBrowse) this would be the first heading. Maybe we should retitle it for now as "Danforth Center genomics pipeline". We should add a separate sections or pages for the GBS/RIL analysis done by KSU and the CoGe pipeline (which should be very similar or essentially the same as Phil's).

Reply:
* KSU pipeline stub: https://github.com/terraref/documentation/pull/170
* CoGe pipeline stub: https://github.com/terraref/documentation/pull/170